easybuild/easyconfigs/r/RevBayes/RevBayes-1.1.1-foss-2019b.eb

```diff
@@ -0,0 +1,46 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Helena Gomez <helena.gomez@hpcnow.com>
+# License::     GPL-v3.0
+# #
+easyblock = 'CmdCp'
+
+name = 'RevBayes'
+version = '1.1.1'
+
+homepage = 'https://revbayes.github.io/'
+
+description = """RevBayes provides an interactive environment for statistical
+ computation in phylogenetics. It is primarily intended for modeling,
+ simulation, and Bayesian inference in evolutionary biology, particularly
+ phylogenetics."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://github.com/revbayes/revbayes/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['d61293fceac817d8203ed1e828661d76c73fa16bf04458a50a37057e99fd40c0']
+
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+    ('Boost', '1.71.0'),
```

Comment:
@YensyGomez It doesn't make sense to include `Boost` as a dependency twice, only via `dependencies` is fine (unless `Boost` is only a build dependency, then only via `builddependencies` of course)